package.json

```diff
@@ -16,6 +16,7 @@
     "cs-check": "lerna run cs-check",
     "cs-format": "lerna run cs-format",
     "build": "lerna run --stream build",
+    "build-serial": "lerna run --concurrency 1 --stream build",
```

Comment:
@ranihorev Can you please also update the `contributing.md` file to provide this as an alternate solution?

Reply:
Done, lmk what you think